libadikted/Makefile

```diff
@@ -1,14 +1,51 @@
 # Project: libadikted
+# Makefile created by KeeperFX/dkfans team
 
-#To create a debug executable, run with the command:
-# make DEBUG
+# To create a debug executable, run with the command:
+# make DEBUG (THIS IS NOT IMPLEMENTED)
 
-#CROSS_COMPILE = i686-w64-mingw32-
+# Instructions:
+# To create adikted.dll, run:
+# make all
+# To create adikited.so, run:
+# make all BUILD_FOR_LINUX=1 (not fully implemented)
+# To clean up the built files (objects, library, and resource file) run:
+# make clean
 
+# Tested and working with:
+# - MSYS2 (MinGW 32-bit) - make
+# - MSYS2 (MinGW 32-bit) - mingw32-make
+# - MinGW (MSYS) - make
 
-CC           = $(CROSS_COMPILE)gcc
-LINK         = $(CROSS_COMPILE)gcc
-SRC          = \
+# Executable files extension on host environment
+ifneq (,$(findstring Windows,$(OS)))
+  CROSS_EXEEXT = .exe
+else
+  CROSS_EXEEXT =
+  CROSS_COMPILE = i686-w64-mingw32-
```

Comment:
Just so I can understand. Why do this? Is it simply to have it compile with gcc 10.2 instead of 9.3?

Reply:
It may not be needed, I haven't actually tried to build in Linux yet. @SimLV Sim was playing with it yesterday, before giving up and using Windows. It may just be because Sim was trying to get in line with Adam and myself who are/were using that version of make.

Reply:
Same as with the above, I have the IF the wrong way.

I'll need to retest.

Reply:
Unless standard GCC is able to cross compile to windows, you will need to reference mingw on Linux.